ext/node/ops/crypto/mod.rs

```diff
@@ -419,6 +435,25 @@ pub fn op_node_sign(
         .into(),
       )
     }
+    // signature structure encoding is DER by default for DSA and ECDSA.
+    //
+    // TODO(@littledivy): Validate public_key if present
```

Comment:
Can you open an issue about it?

Reply:
https://github.com/denoland/deno/issues/22919